packages/web/src/components/molecules/TestActionsDropdown/TestActionsDropdown.tsx

```diff
@@ -0,0 +1,112 @@
+import {useCallback, useMemo, useRef, useState} from 'react';
+
+import {ItemType} from 'antd/lib/menu/hooks/useItems';
+
+import {useDashboardNavigate} from '@hooks/useDashboardNavigate';
+import useInViewport from '@hooks/useInViewport';
+import useRunEntity from '@hooks/useRunEntity';
+import {SystemAccess, useSystemAccess} from '@hooks/useSystemAccess';
+
+import {ActionsDropdown} from '@models/actionsDropdown';
+import {ExecutionMetrics} from '@models/metrics';
+
+import DotsDropdown from '@molecules/DotsDropdown';
+import {notificationCall} from '@molecules/Notification';
+
+import {useAbortAllTestExecutionsMutation, useGetTestExecutionMetricsQuery} from '@services/tests';
+
+import {PollingIntervals} from '@src/utils/numbers';
+
+const TestActionsDropdown: React.FC<ActionsDropdown> = props => {
+  const {name, namespace, outOfSync, type} = props;
+
+  const [abortAllTestExecutions] = useAbortAllTestExecutionsMutation();
+
+  const isSystemAvailable = useSystemAccess(SystemAccess.system);
+
+  const [open, setOpen] = useState(false);
+
+  const editNavigate = useDashboardNavigate(() => `/tests/${name}/settings/test`);
+
+  const [, run] = useRunEntity('tests', {
+    name,
+    namespace,
+    type,
+  });
+
+  const ref = useRef(null);
+  const isInViewport = useInViewport(ref);
+
+  const {data: metrics, refetch} = useGetTestExecutionMetricsQuery(
+    {id: name, last: 7, limit: 13},
+    {skip: !isInViewport || !isSystemAvailable, pollingInterval: PollingIntervals.halfMin}
+  );
+
+  const executions: ExecutionMetrics[] = useMemo(() => metrics?.executions ?? [], [metrics]);
+
+  const onAbort = useCallback(() => {
+    setOpen(false);
+
+    abortAllTestExecutions({id: name})
+      .unwrap()
+      .then(() => {
+        refetch();
+      })
+      .catch(() => {
+        notificationCall('failed', 'Something went wrong during test execution abortion');
+      });
+  }, [abortAllTestExecutions, name, refetch]);
+
+  const dropdownItems: ItemType[] = useMemo(
+    () => [
+      {
+        key: `run-test`,
+        label: 'Run test',
+        onClick: async ({domEvent}) => {
+          domEvent.stopPropagation();
```

Comment:
Isn't `event.stopPropagation()` working as well, instead of `event.domEvent.stopPropagation()`? Same for others

Reply:
This is what we get from the antd component, event is not found as type

Reply:
Checked, and this is passed here:
```ts
export interface MenuInfo {
    key: string;
    keyPath: string[];
    /** @deprecated This will not support in future. You should avoid to use this */
    item: React.ReactInstance;
    domEvent: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>;
}
```

Resolving then 👍